Assets/TilemapHeightTest/Scripts/TileHeightManager.cs

```diff
@@ -41,7 +41,11 @@ public class TileHeightManager : MonoBehaviour
 
 		private void Awake()
 		{
-			Instance = this;
+			if (Instances == null)
+            {
```

Comment:
Nit: Use tabs instead of spaces (not because tabs are better than spaces, but because all other lines are using tabs)